.github/ISSUE_TEMPLATE/bug-fr.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Bogue
+about: Générer un rapport de bogue pour nous aider à nous améliorer
+title: "[BUG] Nom de la fonctionnalité - Courte description du bogue"
+labels: ''
+assignees: ''
+
+---
+
+## Information nécessaire
+
+**Description du bogue**
+Une description claire et concise du bogue.
+Si vous avez trouvé un autre rapport de bogue relié à ceci sur GitHub, veuillez mentionner son numéro ex. : #7890
+
+**Pour le reproduire**
```

Comment:
```suggestion
**Pour le reproduire**
<!-- Obligatoire -->
```

Reply:
Having it in the preceding section header is sufficient.